### PR TITLE
Removed artifacts feature

### PR DESCRIPTION
**Pull request by Raalsky**
## Before submitting checklist

- [x] Did you **update the CHANGELOG**? (not for test updates, internal changes/refactors or CI/CD setup)
- [ ] Did you **ask the docs owner** to review all the user-facing changes?

### CODE REVIEW

File: .github/workflows/e2e.yml
[the remark this answers is not in the document]
Comment (Raalsky) [reply .github/workflows/e2e.yml:1]: Yes, artifacts tracks files on selected storage (local, or AWS S3-compatible including GCS).

File: pyproject.toml
[the remark this answers is not in the document]
Comment (Raalsky) [reply pyproject.toml:1]: Ohh, my fault. That was for debugging purposes.

File: src/neptune/core/typing/container_type.py
[the remark this answers is not in the document]
Comment (Raalsky) [reply src/neptune/core/typing/container_type.py:1]: Nothing, done by occasion but I'll revert it for now.

File: pyproject.toml
[the remark this answers is not in the document]
Comment (Raalsky) [reply pyproject.toml:1]: Done

File: src/neptune/core/typing/container_type.py
[the remark this answers is not in the document]
Comment (Raalsky) [reply src/neptune/core/typing/container_type.py:1]: Done

**Review by szysad — APPROVED** [review]

**Pull request merged**